netboot/main.go

```diff
@@ -174,6 +175,7 @@ func boot(ifname string, dhcp dhcpFunc) error {
 		log.Printf("netboot: attempt %d for http.Get", attempt+1)
 		resp, err = http.Get(bootfile)
 		if err != nil && retryableNetError(err) || retryableHTTPError(resp) {
+			time.Sleep(retryInterval)
```

Comment:
Should the delay be increased after each retry?

Reply:
yes, it's part of https://github.com/systemboot/systemboot/issues/100 , and the reason why I didn't implement retry on 503 to avoid hammering the remote servers until exponential backoff is there. With timeouts and 500/502 I'm assuming it's safer to retry without backoff